push-images.sh

```diff
@@ -11,21 +11,10 @@ else
 fi
 
 TAG_NAME="${SELF_VERSION}_${SPARK_LABEL}_hadoop-${HADOOP_VERSION}_scala-${SCALA_VERSION}_java-${JAVA_VERSION}"
-ALT_TAG_NAME="${SPARK_LABEL}_hadoop-${HADOOP_VERSION}_scala-${SCALA_VERSION}_java-${JAVA_VERSION}"
 
-docker tag "${IMAGE_NAME}:${TAG_NAME}" "${IMAGE_ORG}/${IMAGE_NAME}:${TAG_NAME}"
+docker tag "${IMAGE_NAME}:${TAG_NAME}" "test_${IMAGE_ORG}/${IMAGE_NAME}:${TAG_NAME}"
```

Comment:
do you need to revert `test_` prefix?

Reply:
ah thanks for the catch [6c00c0e](https://github.com/dsaidgovsg/spark-k8s/pull/42/commits/6c00c0e0e96b815d57bf32c939f2e3079fab3305)